examples/multi_script_CLI.py

```diff
@@ -1,97 +1,607 @@
+            print("Delay from one comment to another comment you will perform (enter to use default number: 60)")
+            f.write(str(int(sys.stdin.readline().strip()or "60")) + "\n")
+            print("Want to use proxy? insert your proxy or leave it blank if no. (just enter)")
+            f.write(str(str(sys.stdin.readline().strip()or "None")) + "\n")
+            print("done with all settings")
+            break
 
+def parameter_setting():
+    print("current parameter\n")
+    f=open(setting)
+    data=f.readlines()
+    print("Max likes per day: "+data[0])
+    print("Max unlikes per day: "+data[1])
+    print("Max follows per day: "+data[2])
+    print("Max unfollows per day: "+data[3])
+    print("Max comments per day: "+data[4])
+    print("Max likes to like: "+data[5])
+    print("Max followers to follow: "+data[6])
+    print("Min followers to follow: "+data[7])
+    print("Max following to follow: "+data[8])
+    print("Min following to follow: "+data[9])
+    print("Max followers to following_ratio: "+data[10])
+    print("Max following to followers_ratio: "+data[11])
+    print("Min media_count to follow:"+data[12])
+    print("Like delay: "+data[13])
+    print("Unlike delay: "+data[14])
+    print("Follow delay: "+data[15])
+    print("Unfollow delay: "+data[16])
+    print("Comment delay: "+data[17])
+    print("Proxy: "+data[18])
 
+def username_adder():
+    with open(SECRET_FILE, "a") as f:
+        print("we will add your instagram account.")
+        print("Don't worry. It will be stored locally.")
+        while True:
+            print("Enter your login: ")
+            f.write(str(sys.stdin.readline().strip()) + ":")
+            print("Enter your password: ")
+            f.write(getpass.getpass() + "\n")
+            print("Do you want to add another account? (y/n)")
+            if "y" not in sys.stdin.readline():
+                break
+
+def hashtag_adder():
+    print("current database:")
+    print(bot.read_list_from_file(hashtag_file))
+    with open(hashtag_file, "a") as f:
+        print('Add hashtag to database')
+        while True:
+            print("Enter hashtag: ")
+            f.write(str(sys.stdin.readline().strip()) + "\n")
+            print("Do you want to add another hashtag? (y/n)")
+            if "y" not in sys.stdin.readline():
+                print('done adding hashtag to database')
+                break
+
+def competitor_adder():
+    print("current database:")
+    print(bot.read_list_from_file(users_file))
+    with open(users_file, "a") as f:
+        print('Add username to database')
+        while True:
+            print("Enter username: ")
+            f.write(str(sys.stdin.readline().strip()) + "\n")
+            print("Do you want to add another username? (y/n)")
+            if "y" not in sys.stdin.readline():
+                print('done adding username to database')
+                break
+
+def blacklist_adder():
```

Comment:
doubling the code - make one function with arguments (file_name) instead of several equal.